### ISSUE TITLE
Entity Management of "Invoice" not possible

### ISSUE THREAD

**Issue by clicktrend** [marellocommerce/marello#63]
Thrown error

    Unable to generate a URL for the named route "marello_invoice_index" as such route does not exist.

I guess it should be "marello_invoice_view" like in line 19 in file InvoiceController.php

**Comment by 24198** [comment]
Hi @clicktrend,

Thanks for the report. The route "marello_invoice_index" should actually be "marello_invoice_invoice_index", we noticed that we haven't changed the entity config (in the Marello\Bundle\InvoiceBundle\Entity\AbstractInvoice) which is causing issues with the entity management part of the application specifically for the Invoice. We were able to confirm this issue and will fix it for the next release.

Thanks again!

Jaimy
